bsp/spike/metal-inline.h

```diff
@@ -0,0 +1,139 @@
+/* Copyright 2019 SiFive, Inc */
+/* SPDX-License-Identifier: Apache-2.0 */
+/* ----------------------------------- */
+/* ----------------------------------- */
+
+#ifndef ASSEMBLY
+
+#ifndef METAL_INLINE_H
```

Comment:
Just occurred to me, I think we should also emit the target name in these generate files to, since all metal files name will be the same.